core/trino-main/src/main/java/io/trino/sql/planner/SplitSourceFactory.java

```diff
@@ -20,9 +20,9 @@
 import io.trino.Session;
 import io.trino.metadata.Metadata;
 import io.trino.server.DynamicFilterService;
```

Comment:
what do you pass now instead of `ALL`? Should we just deprecate `getSplits` method that accepts constraint (predicates should be pushed by `applyFilter`)

Reply:
Thanks for explanation. So what do you pass now instead of ALL?